tests/integration/test_profile_chunks.py

```diff
@@ -215,3 +232,97 @@ def test_profile_chunk_outcomes_rate_limited(
         outcome.pop("event_id", None)
 
     assert outcomes == expected_outcomes, outcomes
+
+    # Verify no profiles were forwarded to the consumer
+    profiles_consumer.assert_empty()
+
+
+def test_profile_chunk_outcomes_rate_limited_via_profile_duration_rate_limit(
+    mini_sentry,
+    relay_with_processing,
+    outcomes_consumer,
+    profiles_consumer,
+):
+    """
+    Tests that Relay reports correct outcomes when profile chunks are rate limited via profile duration quotas.
+
+    This test verifies that when a profile chunk hits a profile duration rate limit:
+    1. The profile chunk is dropped and not forwarded to the profiles consumer
+    2. A rate limited outcome is emitted with the correct category and reason code
+    3. The rate limit is enforced at the organization level
+    4. Both profile_chunk and profile_duration categories are affected
+    """
+    outcomes_consumer = outcomes_consumer(timeout=2)
+    profiles_consumer = profiles_consumer()
+
+    project_id = 42
+    project_config = mini_sentry.add_full_project_config(project_id)["config"]
+
+    # Enable profiling feature flag
+    project_config.setdefault("features", []).append(
+        "organizations:continuous-profiling"
+    )
+
+    # Configure rate limiting quota that blocks all profile durations and profile chunks at org level
+    project_config["quotas"] = [
+        {
+            "categories": ["profile_duration", "profile_chunk"],
```

Comment:
Would this test also pass with only `"categories": ["profile_duration"]`?

Reply:
This was for completeness as this would be the quota confi we would send from `getsentry`: https://github.com/getsentry/getsentry/pull/15734